src/components/PartnersContent/PartnersContent.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FunctionComponent, useState, useEffect } from "react";
+import { PartnersTile } from "./PartnersTile";
+import { Pagination } from "@govtechsg/tradetrust-ui-components";
+import { PartnerDataSingle, PartnersDataProps } from "./types";
+
+const MAX_PARTNER_TILES_TO_DISPLAY = 9;
+
+export const sortPartnersData = (allPartnersData: PartnerDataSingle[]): PartnerDataSingle[] => {
+  return allPartnersData.sort((a, b) => a.attributes.companyName.localeCompare(b.attributes.companyName)); // this sort it alphabetically by companyName
+};
+
+export const PartnersContent: FunctionComponent<PartnersDataProps> = ({ partnersData }) => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [pageData, setPageData] = useState<PartnerDataSingle[]>([]);
+  const [totalNoOfPages, setTotalNoOfPages] = useState(1);
+
+  useEffect(() => {
+    const sortedData = sortPartnersData(partnersData);
+    setTotalNoOfPages(Math.ceil(sortedData.length / MAX_PARTNER_TILES_TO_DISPLAY));
```

Comment:
shift `sortedData` all the way out to `Partners.tsx` and pass as prop, so it wont be sorted again over and over again on every `currentPage` update

Reply:
ok got it